jar-infer/jar-infer-lib/src/main/java/com/uber/nullaway/jarinfer/DefinitelyDerefedParamsDriver.java

```diff
@@ -393,7 +384,7 @@ private static void writeModel(DataOutputStream out) throws IOException {
     StubxWriter.write(out, importedAnnotations, packageAnnotations, typeAnnotations, methodRecords);
   }
 
-  private static void WriteAnnotations(String inPath, String outPath) throws IOException {
+  private void WriteAnnotations(String inPath, String outPath) throws IOException {
```

Comment:
let's take this opportunity to switch this method to lowercase :)

Reply:
Done.